features/minion/heartbeat/consumer/src/main/java/org/opennms/minion/heartbeat/consumer/HeartbeatConsumer.java

```diff
@@ -154,7 +154,7 @@ private void provision(final OnmsMinion minion,
             return;
         }
 
-        // Return fast until the provisioner is running to pick up the events send below
+        // Return fast until the provisioner is running to pick up the events sent below
         if (!this.eventSubscriptionService.hasEventListener(EventConstants.RELOAD_IMPORT_UEI)) {
```

Comment:
Should we also attempt to catch the *IllegalStateException* here, and prevent this known case from appearing in the logs?